internal/reader/home_chain.go

```diff
@@ -322,6 +413,67 @@ func convertOnChainConfigToHomeChainConfig(
 	return chainConfigs
 }
 
+func convertOnChainConfigToRMNHomeChainConfig(
+	lggr logger.Logger,
+	versionedConfigWithDigests []VersionedConfigWithDigest,
+) map[cciptypes.Bytes32]rmn.RMNHomeConfig {
+	if len(versionedConfigWithDigests) == 0 {
+		lggr.Warnw("no on chain RMNHomeConfigs found")
+		return map[cciptypes.Bytes32]rmn.RMNHomeConfig{}
+	}
+
+	rmnHomeConfigs := make(map[cciptypes.Bytes32]rmn.RMNHomeConfig)
+	for _, versionedConfigWithDigest := range versionedConfigWithDigests {
+		config := versionedConfigWithDigest.VersionedConfig.Config
+		nodes := make([]rmn.RMNHomeNodeInfo, len(config.Nodes))
+		for i, node := range config.Nodes {
+			pubKey := ed25519.PublicKey(node.OffchainPublicKey[:])
+
+			nodes[i] = rmn.RMNHomeNodeInfo{
+				ID:                        rmn.NodeID(i),
+				PeerID:                    node.PeerID,
+				SignObservationsPublicKey: &pubKey,
+				SupportedSourceChains:     mapset.NewSet[cciptypes.ChainSelector](),
+			}
+		}
+
+		minObservers := make(map[cciptypes.ChainSelector]uint64)
+
+		for _, chain := range config.SourceChains {
+			minObservers[chain.ChainSelector] = chain.MinObservers
+			for j := 0; j < 256; j++ {
```

Comment:
what's the 256?

Reply:
Max RMN committee size, should be pulled into a const